plugins/mac-avcapture/OBSAVCapturePresetInfo.h

```diff
@@ -0,0 +1,24 @@
+//
+//  OBSAVCapturePresetInfo.h
+//  mac-avcapture
+//
+//  Created by Patrick Heyer on 07.03.23.
```

Comment:
Could we make the date less ambiguous by using an ISO 8601 date (YYYY-MM-DD), here and elsewhere?

Reply:
Sure, it was autogenerated by Xcode so I didn't pay much attention.